### PR TITLE
Feature/components

### PR DESCRIPTION
**Pull request by tnyandoro**
To help you concentrate on how to work with React events, we have developed the math logic you need to make the Calculator component work. You can use the files calculate.js and operate.js.

Copy the files calculator.js and operate.js into a logic/ directory in your project.
Analyze the files calculator.js and operate.js, be sure you understand what they do.
Import the files in your Calculator component.
Implement the event handlers you need to use the math logic from calculator.js and operate.js in your React component.
Practice what you have learned until now: use state and props, pass props to child components, use lifecycle methods, lift state up.
Test your app in the browser. It should be able to perform math operations.

### CODE REVIEW

File: src/assets/button.module.css [hunk 1]
```diff
@@ -0,0 +1,31 @@
+.Button {
+  width: 25%;
+  height: 100%;
+  font-size: 3rem;
+  font-family: inherit;
+  background: #cad2c5;
+  border: 1px solid transparent;
```
Comment (henatan99) [inline src/assets/button.module.css]: - [x] To fit with the original design requirement, kindly avoid the `transparent` property of the buttons. It needs a visible border as you can compare it with the original design. 
- [x] The button text looks a bit extra big. It doesn't achieve a good size contrast and affects the user experience. Kindly, follow the original design. 
- [x] OPTIONAL: While it still looks ok, you can still make changes the background colors to meet the original design requirements. 
image
image

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi @tnyandoro , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
- [x] Your PR summary is a summary of the changes you made in this milestone
- [x] Kindly refactor your `App` component to React class component as it is this Milestone's objective to practice React class components. 
- [x] Please update your readme. While the default readme gives some general info about react-app you should add a slight description of your project. You are highly encouraged to use the Microverse [readme template](https://github.com/microverseinc/readme-template). 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/assets/button.module.css (hunk 1 above)
Comment (tnyandoro) [reply src/assets/button.module.css]: Resolved

**Comment by tnyandoro** [comment]
- Created a directory called components.
- Inside components, created a new Calculator.js file.
- In Calculator.js, create a React component that matches this design:
  Calculator


 - eveloped the UI of the calculator; the math logic to make it work will be added in the next project.
- Used class based components.
- Once the Calculator is ready, import it and use it in your main component (App.js).
- Tested it in the browser; it should render without problems.

**Review by henatan99 — APPROVED** [review]
Hi @tnyandoro , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

- [ ] OPTIONAL: YOu don't need to use another comment box for the PR summary. You could have edited the original. Just take this point not to make the same mistake in your future PRs. The changes you made are great though! :clap: 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**